### ISSUE TITLE
Add different container options as utility class

### ISSUE THREAD

**Issue by Sidstumple** [fdnd/components#2]
There are several containers/borders possible for blocks, f.e.:

![Image](https://user-images.githubusercontent.com/14167636/195583823-be3a7063-2fa9-4f25-b8bd-fb734de68e82.png)

![Image](https://user-images.githubusercontent.com/14167636/195583838-546722ca-3bef-454d-abd2-5e6a45b7aaab.png)

![Image](https://user-images.githubusercontent.com/14167636/195583957-052f4b7e-6f21-427f-a5da-c036623a63a8.png)

![Image](https://user-images.githubusercontent.com/14167636/195583850-faed4ea2-18c5-40a7-bdc5-6558fee5a759.png)

Color of the border/container should also be editable with a utility class:

![Image](https://user-images.githubusercontent.com/14167636/195583884-ab063a08-3e7b-44b8-94de-f838942525e7.png)

## Naming
We should determine clear naming for this; my first idea is:

`border-clip--top-right border-color--purple`
`border-clip--top-right border-color--teal`

**Comment by ju5tu5** [comment]
I agree with the naming pattern. To add a layer of complexity, the border color depends on the color of the field. In the above example with the green field and green shadow, the border should be blue (as it is in the example). Do we build in these options automagically? Do we want the option to do really weird things like white field with purple border and green shadow?
